utils/execTask.js

```diff
@@ -1,13 +1,27 @@
 const execa = require("execa");
 const dedent = require("dedent");
+const chalk = require("chalk");
 const symbols = require("log-symbols");
 const npmWhich = require("npm-which")(process.cwd());
 
-module.exports = function execTask({ command, fileList }) {
+function getFormattedTime(end) {
+  return Math.round((end[0] * 1000) + (end[1] / 1000000));
+}
+
+module.exports = function execTask({ command, fileList, task }) { // eslint-disable-line
```

Comment:
I don't think disabling eslint is not needed here 🤔

Reply:
Good catch, I have removed this. I also updated package.json to run eslint for files present in other folders. Earlier, it was being run only for files that were at the same depth as package.json.

Reply:
Great 😃